.github/workflows/nodejs-license.yaml

```diff
@@ -19,8 +19,9 @@ env:
     Public Domain;
     WTFPL;
     Unlicense;
+    BlueOak-1.0.0;
   # It has to be one line, there must be no space between packages.
-  EXCLUDE_PACKAGES: testrpc@0.0.1;uuid@2.0.1;
+  EXCLUDE_PACKAGES: testrpc@0.0.1;uuid@2.0.1;@cspell/dict-en-common-misspellings@2.0.0;
```

Comment:
@Deniallugo had to exclude the package rather then include it in the allowed packages as the license checker does not recognize `CC BY-SA 4.0;`. It should be `CC-BY-SA-4.0;` but `@cspell/dict-en-common-misspellings@2.0.0;` has it misspelled in their `package.json`.

Reply:
Ok, both are good :)